src/cgtop/cgtop.c

```diff
@@ -852,7 +852,7 @@ static int parse_argv(int argc, char *argv[]) {
                         }
 
                         arg_recursive = r;
-                        recursive_unset = r == 0;
+                        arg_recursive_unset = r == 0;
```

Comment:
hmm, why do we need both arg_recursive and arg_recursive_unset as public fields? Not following? Wouldn't one suffice?